packages/tools/pie-icons/src/assets/qr-code-large.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If it is a PIE icon it should have size 32x32 for large and 16x16 for small icons